### DISCUSSION TITLE
Terminal

### DISCUSSION THREAD

**Discussion by diapoio** [balta-io/2801#26] [Perguntas e Respostas] [answered]
Balta Feliz 2022, começei o curso essa semana, o commando >shell command nao aparece no vccode. vi que tem uma opção no menu de terminal, porem nao esta funcionando o commando dotnet new console.

**Comment by diapoio** [comment, marked as answer]
ja resolvi, mas nao consegui excluir a pergunta

**Reply by andrebaltieri** [reply]
Perfeito 💜

**Comment by robsonmagno12** [comment]
O meu está dando erro no dotnet restore


C:\Program Files\dotnet\sdk\6.0.101\Current\SolutionFile\ImportAfter\Microsoft.NET.Sdk.Solution.targets(14,3): error MSB4019: The imported project "C:\Program Files\dotnet\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.Common.targets" was not found. Confirm that the expression in the Import declaration "C:\Program Files\dotnet\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.Common.targets" is correct, and that the file exists on disk.

**Reply by andrebaltieri** [reply]
Consegue mandar um print com o projeto aberto? Este erro ocorre onde e quando?

**Comment by robsonmagno12** [comment]
Conseguir rodar no Visual Studio baixado ls. mas só no VS consigo.
Encaminhado quando tento rodar no terminal. Mas fui testar novamente tanto
flutter restore, build, run não ler
[image: image.png]

Em seg., 17 de jan. de 2022 às 13:10, Andre Baltieri <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
@robsonmagno12 acho que você está respondendo o E-mail do GitHub, anexa a imagem pelo site (github.com) por favor... não veio nada.
Sua resposta ficou bem confusa, dá uma olhada por favor!

**Comment by robsonmagno12** [comment]
Ontem estava rodando normalmente no VS

Em seg., 17 de jan. de 2022 às 14:33, robson magno ***@***.***>
escreveu:

**Comment by robsonmagno12** [comment]
Ok!

**Comment by robsonmagno12** [comment]
Conseguir professor era o Path. Achei estranho já que crio API's e manutenção e dando problema simples em rodar no terminal.

**Reply by andrebaltieri** [reply]
Perfeito!!

**Comment by ale-fuentes** [comment]
Estimado @andrebaltieri , boa tarde.
No video sobre CLI, fala de na descrição do video estaria sobre o layout do Windows Terminal, mas eu não acho as descrições.
O senhor teria como encaminhar ?

Grato pela ajuda

**Reply by andrebaltieri** [reply]
https://balta.io/blog/windows-terminal
